solutions/security/get-started/get-started-detect-with-siem.md

```diff
@@ -0,0 +1,135 @@
+---
+navigation_title: Detect and respond to threats with SIEM
+description: An introduction to detecting threats with SIEM in {{elastic-sec}}.
+applies_to:
+  serverless:
+products:
+  - id: security
+---
+
+# Quickstart: Detect and respond to threats with SIEM
+
+Elastic Security is a unified security solution that brings together SIEM, endpoint security, and cloud security into a single platform. This makes it easier to protect, investigate, and respond to security events from all areas within your environment.
+
+In this guide, we'll learn how to use some of {{elastic-sec}}'s SIEM features to detect, investigate, and respond to threats. 
+
+## Prerequisites 
+
+To get started exploring {{elastic-sec}}, log in to your {{sec-serverless}} project. If you don't have one yet, refer to [Create a Security project](/solutions/security/get-started/create-security-project.md). 
+
+:::::{stepper}
+
+::::{step} Add data using Elastic Defend
+
+Before you can start using {{elastic-sec}}, you need to choose an integration to start collecting and analyzing your data. For this guide, we're going to use the {{elastic-defend}} integration. {{elastic-defend}} detects and protects endpoints from malicious activity and provides automated response options before damage and loss occur. 
+
+1. On the Get started home page, in the **Ingest your data** section, select **Elastic Defend**, then click **Add Elastic Defend**. 
```

Comment:
Just leaving a single styling comment in case you decide against bolding the page name across the page.

```suggestion
1. On the **Get started** page, in the **Ingest your data** section, select **Elastic Defend**, then click **Add Elastic Defend**. 
```